dataikuapi/dssclient.py

```diff
@@ -667,6 +670,105 @@ def create_global_api_key(self, label=None, description=None, admin=False):
         key = resp.get('key', '')
         return DSSGlobalApiKey(self, key)
 
+    ########################################################
+    # Personal API Keys
+    ########################################################
+
+    def list_personal_api_keys(self, as_type='object'):
```

Comment:
Careful : mismatch between `object` and `objects` (also in the doc).
So I did not test but if we follow the doc and/or use the default value you'll get a `dict`.

Reply:
Good catch!